addon/components/tags-widget/component.js

```diff
@@ -49,22 +49,28 @@ export default Ember.Component.extend({
                     this.send('removeATag', tag);
                 }
             });
-            // Populate the widget with the default tags passed in
-            let tags = this.get('tags');
-            this.$().importTags(tags.join(', '));
         });
     }),
 
+    didRender() {
+        // Rerender the list of tags whenever the node model changes. Useful if node.tags is not defined when page loads.
+        // Provide a default value in case tags weren't defined when component first rendered
+        this.$().importTags('');
```

Comment:
As discussed last week- when this was hooked up to an observer, it caused unit tests to fail.